037-WindowsVirtualDesktop/Student/01-Plan-WVD-Architecture.md

```diff
@@ -0,0 +1,60 @@
+# Challenge 1: Design the Windows Virtual Desktop architecture 
+
+[Previous Challenge](./00-Pre-Reqs.md) - **[Home](../README.md)** - [Next Challenge>](./02-Implement-Manage-Network.md)
+
+
+## Introduction 
+
+This Challenge is to design the WVD architecture that will be deployed in later challenges.  By the end of this session you should have have a plan for the host pools, session hosts, profile storage and client deployment.  
+
+You should also have considered all aspects of security in this design including GPO, identity, and RBAC.
+
+## Description
+
+Taking into account our scenario, specify the details for the following:
+
+1. Host pool(s)  
+1. Images
+1. Storage  
+1. Clients
+1. Security  
+1. Networking
+1. FSLogix
+
+
+## Success Criteria
+
+This Challenge will be succesfully completed when you have created a document defining all of the parameters that will be used in the deployment challenges.  
```

Comment:
I would change this from a sentence to a bullet list, even if it is a list of 1, such as:

- You have created a document defining all parameters used in the deployment challenges.